transient/editor.py

```diff
@@ -170,14 +170,14 @@ def _mount_root(self) -> None:
             try:
                 self.run_command_in_guest(
                     [
-                        f"mount -t {fstype} {path} /mnt > /dev/null",
+                        f"mount -t {fstype} {path} /mnt &> /dev/null",
                         "[ -f /mnt/etc/fstab ]",
                     ],
                 )
                 return
             except Exception as e:
                 logging.debug(f"Failed to read /etc/fstab from {path}: {e}")
-                self.run_command_in_guest("umount /mnt > /dev/null", allowfail=True)
+                self.run_command_in_guest("umount /mnt &> /dev/null", allowfail=True)
```

Comment:
I believe `&>` is a bashism. While there may be others in this code base, I'd like to avoid them if possible. Should be fine to write `2>&1 > /dev/null`

Reply:
Changed in my latest push. In testing though, the `2>&1` had to come after the regular redirect instead of before.